src/display/canvas.js

```diff
@@ -856,7 +856,10 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
       if (this.transparentCanvas) {
         this.ctx = this.compositeCtx;
+        this.ctx.save();
```

Comment:
Shall we just call `this.ctx.restore()` here one more time? We have non-pair this.ctx.save() in the beginDrawing method.